4.classify-features/classify-features.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Classify Cell Health Nuclei Features
+# 
+# ### Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import numpy as np
+import urllib.request
+import joblib
+
+import importlib
+classification_utils = importlib.import_module("classification-utils")
+
+
+# ### Download/load `phenotypic_profiling` model
+
+# In[2]:
+
+
+hash = "64cfc46ecd92f1956af199c81f8ecf4dc292718f"
```

Comment:
[`hash`](https://docs.python.org/3/library/functions.html?highlight=hash#hash) is a Python built-in function. Consider renaming this variable to avoid an overlap, for ex:
```suggestion
gh_hash = "64cfc46ecd92f1956af199c81f8ecf4dc292718f"
```